### DISCUSSION TITLE
Login problems while trying to clone a Git Repository using STS4

### DISCUSSION THREAD

**Discussion by philfrei** [community/community#22402] [New to GitHub]
<p>I am using Sprint Tool Suite 4 (Eclipse), which has a “github” view.<br>
I have a project based on a local clone of a github project. This local clone reflects the contents of the project at the time I originally cloned it (months ago). I was hoping to update the local clone, but have been unable to figure out how to do this.</p>
<p>I thought it might be easier to instead start over. I now have a fork of the project on my own account. Using the option “Clone a Git repository” I specified the address (yes, included .git), and put in my id and password. But it seems I’m not allowed to connect.</p>
<p>The error message includes the following as a possible cause:</p>
<blockquote>
<p>SSL host could not be verified (set http.sslVerify=false in Git configuration)</p>
</blockquote>
<p>I’m not using SSL host, am I? Is that something that is happening “under the hood”? Is the normal signin name and password no longer sufficient for this operation?</p>

**Comment by airtower-luna** [comment]
<p>Well, assuming you’re using the HTTPS URL, that means HTTP over TLS. SSL was the predecessor of TLS, and some error messages never got updated. :wink:</p>
<p>The message sounds like the host certificate couldn’t be verified, that means <em>something</em> is interfering with the secure connection to GitHub. What or why is hard to say from the outside, could be any of the following and probably a couple other things:</p>
<ul>
<li>An outdated/misconfigured list of trusted CAs on your computer.</li>
<li>A badly configured firewall or antivirus. Some vendors/companies are of the misguided opinion that breaking TLS encryption to inspect the content would increase security.</li>
<li>Some sort of network problem that sends your connection to the wrong place.</li>
<li>Even a virus or hack.</li>
</ul>
<p>Disabling certificate verification is a really bad suggestion there because you’d end up sending your GitHub credentials to whatever is interfering with the connection. If it’s something malicious that thing might even forward them (and responses) so things look like they’re working and it can keep stealing your data.</p>
<p>The best thing to do here would be to investigate where exactly the problem happens. A good starting points are: Can other software (e.g. your browser) reach GitHub from the same system, and verify the certificate? Does the same thing happen from another network?</p>

**Comment by philfrei** [comment, marked as answer]
<p>Thanks for the reply. I’ve been working on this since posting and have moved forward in my understanding.</p>
<p>I used the following guide to create a <em>personal access token</em>.<br>
<a href="https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token#creating-a-token" rel="noopener nofollow ugc">https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token#creating-a-token</a></p>
<p>Having this in hand, when STS4 asked me for a password, I gave it the p-a-t. This got me to the next screen. I just got there moments ago, and haven’t finished setting up the project yet. But with this, the initial question (log in problem) is answered.</p>
